include/constants/battle_script_commands.h

```diff
@@ -237,6 +237,7 @@
 #define VARIOUS_STORE_HEALING_WISH                   145
 #define VARIOUS_HIT_SWITCH_TARGET_FAILED             146
 #define VARIOUS_TRY_REVIVAL_BLESSING                 147
+#define VARIOUS_TRY_ACTIVATE_GULP_MISSILE            148
```

Comment:
Instead of a various could you use a callnative? For examples you can check all the callnatives at the end of the `battle_script_commands.h` file. 

Generally we try to phase out variouses.